lib/utils/createComponentsFromSoup.ts

```diff
@@ -6,12 +6,21 @@ import { PlatedHole } from "lib/components/primitive-components/PlatedHole"
 import { Keepout } from "lib/components/primitive-components/Keepout"
 import { Hole } from "lib/components/primitive-components/Hole"
 import { SilkscreenText } from "lib/components/primitive-components/SilkscreenText"
+import { createPinrowSilkscreenText } from "./createPinrowSilkscreenText"
+import type { PinLabelsProp } from "@tscircuit/props"
 
 export const createComponentsFromSoup = (
```

Comment:
```suggestion
export const createComponentsFromCircuitJson = (
```

Also, update the filename